src/Command/SyncDeletedCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Command;
+
+use App\Service\LeantimeApiService;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Scheduler\Attribute\AsPeriodicTask;
+
+#[AsCommand(
+    name: 'app:data-providers:sync-deleted',
+    description: 'Sync Data Provider deleted data, that has been deleted within the last hour, as jobs. Scheduled to run every 15 minutes.',
+)]
+#[AsPeriodicTask(frequency: '15 minutes')]
```

Comment:
I havn't tried it, but it should be possible to do something like `#[SomeAttribute('%env(MY_ENV_VAR)%')]`, so

```suggestion
#[AsPeriodicTask(frequency: '%env(SYNC_FREQUENCY)%')]
```

And then set `SYNC_FREQUENCY='15 minutes'` in `.env` so it's configurable per installation